npm/react/cypress/support/index.js

```diff
@@ -15,5 +15,3 @@
 
 // custom commands provided by this package, built from TypeScript code in "lib"
 // using "npm run transpile"
-import '@cypress/react/hooks'
-import 'cypress-react-selector'
```

Comment:
We don't import support right now. (honestly, we shouldn't be running this hook for _every_ test anyways... only the ones that use mount)

I moved these into the top of `mount.ts` which seems more appropriate